inginious/common/tags.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for
+# more information about the licensing of this file.
+
+class Tag:
```

Comment:
A tag could be a tuple (id, name), thus avoiding the unneeded class. `parse_tag_str` could be a simple function available in this file. It would be even better with a function name like `parse_tags_from_string`.

Reply:
I create a class for the tag for extensibility reasons. For example, we could add lots of fields like 'is_shown_to_student' to keep the tag hidden to students (only for admin), 'color' if we want to customize the tag rendering, 'explanations' to get information of the purpose of the tag with a tooltip HTML for example. I think this would be easier to add these things with a class instead of a tuple. What do you think ?

Reply:
Ok then :-)